lib/bn.js

```diff
@@ -459,6 +460,11 @@ BN.prototype.toJSON = function toJSON() {
   return this.toString(16);
 };
 
+BN.prototype.toBuffer = function toBuffer(endian, length) {
```

Comment:
I am not entirely sold that ```toArrayLike(Buffer, ...)``` would be handier to use than ```new Buffer(bn.toArray(...))```.

However BN itself could export an index method and therefore become indexable as an Array?

Or alternatively it could make sense to have a node-bn.js package which has these kind of extensions? Although toBuffer on its own might be too slim for that.